### ISSUE TITLE
error message

### ISSUE THREAD

**Issue by pvdri** [ga-wdi-boston/game-project#118]
still getting error message

client?cd17:5Uncaught TypeError: Cannot read property 'replace' of null(anonymous function) @ client?cd17:5(anonymous function) @ client?cd17:96__webpack_require__ @ bootstrap 33da82d…:50(anonymous function) @ bootstrap 33da82d…:93__webpack_require__ @ bootstrap 33da82d…:50(anonymous function) @ bootstrap 33da82d…:93(anonymous function) @ bootstrap 33da82d…:93

**Comment by RDegnen** [comment]
What are you doing to get this error?

**Comment by pvdri** [comment]
If I knew that I would have fixed it...

**Comment by RDegnen** [comment]
Are you clicking a button? Making an AJAX request?? It isn't enough information to say your just getting an error.

**Comment by pvdri** [comment]
I'm filling out my fields and click submit...

**Comment by RDegnen** [comment]
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=Uncaught+TypeError:+Cannot+read+property+%27replace%27+of+null(anonymous+function)+jquery

Your DOM might not be rendered before jQuery starts trying to interact with elements on the page

**Comment by pvdri** [comment]
so what you recommend as a remedy?

**Comment by RDegnen** [comment]
Make sure all your code is running only after the DOM has been rendered.

**Comment by pvdri** [comment]
how does one make sure that the dom has rendered?

**Comment by pvdri** [comment]
Also, it doesn't sound like that the case because I use jquery to set elements of the dom in the tic tac toe game

**Comment by RealWeeks** [comment]
Can you post your document ready function?

**Comment by pvdri** [comment]
``` js
$(document).ready(function () {
    newGame();
    turn = 1;
    if (turn === 0){
      $("#notice").text(ogo);
    }
    else {
      $("#notice").text(xgo);
    }
});
```

**Comment by RealWeeks** [comment]
What is the newGame function?

**Comment by pvdri** [comment]
``` js
let newGame = function () {
    $('td').one('click', function () {
           if (turn === 0) {
            turn = 1;
            $(this).text(oplayer);
            let x = $(this).attr("id");
            switch (x) {
              case $("#a1").attr("id"):
              arb[0] = 0;
              break;
              case $("#a2").attr("id"):
              arb[1] = 0;
              break;
              case $("#a3").attr("id"):
              arb[2] = 0;
              break;
              case $("#b1").attr("id"):
              arb[3] = 0;
              break;
              case $("#b2").attr("id"):
              arb[4] = 0;
              break;
              case $("#b3").attr("id"):
              arb[5] = 0;
              break;
              case $("#c1").attr("id"):
              arb[6] = 0;
              break;
              case $("#c2").attr("id"):
              arb[7] = 0;
              break;
              case $("#c3").attr("id"):
              arb[8] = 0;
              break;
            }
            winner();
            if($("#notice2").text() === owins)
            {disallow();}
            else if (isFull() !== false)
            {$("#notice").text(nowins);}
            else {$("#notice").text(xgo);}
            }
          else {
            turn = 0;
            $(this).text(xplayer);
            let y = $(this).attr("id");

            switch (y) {
              case $("#a1").attr("id"):
              arb[0] = 1;
              break;
              case $("#a2").attr("id"):
              arb[1] = 1;
              break;
              case $("#a3").attr("id"):
              arb[2] = 1;
              break;
              case $("#b1").attr("id"):
              arb[3] = 1;
              break;
              case $("#b2").attr("id"):
              arb[4] = 1;
              break;
              case $("#b3").attr("id"):
              arb[5] = 1;
              break;
              case $("#c1").attr("id"):
              arb[6] = 1;
              break;
              case $("#c2").attr("id"):
              arb[7] = 1;
              break;
              case $("#c3").attr("id"):
              arb[8] = 1;
              break;
            }
          winner();
          if($("#notice2").text() === xwins)
          {disallow();}
          else if (isFull() !== false)
          {$("#notice").text(nowins);}
          else
          {$("#notice").text(ogo);}
          }
    });
};
```

**Comment by laurenfazah** [comment]
@pvdri it looks like you're assigned to @RDegnen with another issue, but when he's done, I'd love for you to walk me through this `newGame()` function you have.

**Comment by pvdri** [comment]
thank you lauren, ross helped, i think we're good....

**Comment by laurenfazah** [comment]
@pvdri please close this and all other residual open issues.

**Comment by RDegnen** [comment]
Please close
